src/main/java/app/model/hbm/Category.hbm.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<!DOCTYPE hibernate-mapping PUBLIC "-//Hibernate/Hibernate Mapping DTD 3.0//EN"
+                                   "http://hibernate.sourceforge.net/hibernate-mapping-3.0.dtd">
+<!-- Generated Jun 19, 2019 8:07:42 AM by Hibernate Tools 3.5.0.Final -->
+<hibernate-mapping>
```

Comment:
`<hibernate-mapping package="app.model">` ghi rõ  thế này
thì ở class name line bên dưới chỉ cần ` <class name="Category" table="CATEGORY">` và `<one-to-many class="Product"/>` thôi

Reply:
dạ ok ạ